src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__logo-container">
+          <a class="header__link" href="#home">
+            <img src="images/logo.png" alt="Moyo-logo"
```

Comment:
Use correct formatting according to styleguide.
![7712](https://github.com/mate-academy/layout_moyo-header/assets/35075922/bb0fc790-e22a-4053-b6cd-4c58f4e1e97e)